printAll.js

```diff
@@ -0,0 +1,14 @@
+const index = require("./index");
+
+function GetallColorCodedPairs() {
+  var i;
+  var colorCodePair = [];
+  for (i = 1; i < 26; i++) {
```

Comment:
Dont hardcode loop limits. We will mostly forget to change this when we add a color